tests/test_integration/test_suites.py

```diff
@@ -54,15 +49,15 @@ class TestCopy(gt_testing.StencilTestSuite):
 
     dtypes = (np.float_,)
     domain_range = [(1, 25), (1, 25), (1, 25)]
-    backends = CPU_BACKENDS
+    backends = INTERNAL_BACKENDS
     symbols = dict(
         field_a=gt_testing.field(in_range=(-10, 10), boundary=[(0, 0), (0, 0), (0, 0)]),
         field_b=gt_testing.field(in_range=(-10, 10), boundary=[(0, 0), (0, 0), (0, 0)]),
     )
 
     def definition(field_a, field_b):
         with computation(PARALLEL), interval(...):
-            field_b = field_a
+            field_b = field_a  # noqa: F841
```

Comment:
I'd add a short explanation of the flake8 error code:
```suggestion
            field_b = field_a  # noqa: F841  # Local name is assigned to but never used 
```